sdk/script/account-manager.ts

```diff
@@ -9,12 +8,12 @@ var packageJson = require("../../package.json");
 
 declare var fs: any;
 
-if (typeof window === "undefined") {
-    fs = require("fs");
+if (typeof window === `undefined`) {
```

Comment:
I like the idea of using it for everything, but what do you think?

The only tricky thing is with my version of the TS compiler, it doesn't seem to like it when you use template strings with ES6 style import statements.